ngraph/test/runtime/interpreter/evaluates_map.cpp

```diff
@@ -694,7 +694,7 @@ namespace
         {
             if (box_encoding == V5BoxEncoding::CORNER)
             {
-                normalize_corner(boxes, boxes_shape);
+                //normalize_corner(boxes, boxes_shape);
```

Comment:
is `normalize_center` in line 701 supported in NMS op?  
Maybe we should add a function like 'pre-processing'(do the same thing you in the commented function) in the NMS op so we can get the same processing as ref impl without changing the OV, and we can easily turn off the ''pre-processing" if the non pre-processing version is needed in the future.

Reply:
normalize_center is the normal process and in our code now. normalize_corner is a redundant process. I think if we decide to change openvino instead of using IE mode, we can add it in our codes.

Reply:
ok, just for me to understand more, by `redundant ` do you mean  the ref impl result with and without `normalize_corner` are both correct (but different)?

Reply:
Yes, in fact, if we send a box like (9, 0, 0, 5) , the normalize_corner will make it to (0, 0, 9, 5),   this just make the original box to another box. But the normalize_center just convert the data from (x, y, width, height) to (y1, x1, y2, x2), they are still the same box.   So they are different in the field of changing the meaning of the data. That's why i think normalize_corner is weird, it deal with the box it create but not the real box we give it,

Reply:
Thanks, I see. it seems like a bug of OV.